projects/wave-core-new/src/lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component.ts

```diff
@@ -155,6 +155,14 @@ export class MultiLayerSelectionComponent implements ControlValueAccessor, OnCha
         }
     }
 
+    updateLayer(index: number, layer: Layer): void {
```

Comment:
why do you update the selectedLayers on select? wouldn't it suffice to update the layers after submitting the form?